charts/argo-cd/templates/argocd-configs/repository-credentials-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- range $repo_cred_key, $repo_cred_value := .Values.configs.credentialTemplates }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-repo-creds-{{ $repo_cred_key }}
+  labels:
+    argocd.argoproj.io/secret-type: repo-creds
+    {{- include "argo-cd.labels" (dict "context" $ "component" $.Values.server.name "name" "secret") | nindent 4 }}
```

Comment:
I think we can omit `"component" $.Values.server.name "name" "secret"` as this secret is not targeted to one component of ArgoCD.